.github/workflows/ci.yml

```diff
@@ -202,6 +202,9 @@ jobs:
     - name: Rust unit tests
       run:  podman exec agama bash -c "cd /checkout/rust; cargo test --verbose"
 
+    - name: Check rust formatting
+      run:  podman exec agama bash -c "cd /checkout/rust; rustup componet add rustfmt; cargo fmt --all -- --check"
```

Comment:
It looks like there is a typo: It should be "component" (instead of "componet").

Reply:
yeah, I tried it in small spare time at the end of night shift and make a typo. Will try fixed version now.